src/libraries/System.IO.Compression/src/System/IO/Compression/GZipStream.cs

```diff
@@ -32,6 +32,18 @@ public GZipStream(Stream stream, CompressionLevel compressionLevel, bool leaveOp
             _deflateStream = new DeflateStream(stream, compressionLevel, leaveOpen, ZLibNative.GZip_DefaultWindowBits);
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="GZipStream"/> class by using the specified stream, compression options, and whether to leave the <paramref name="stream"/> open.
```

Comment:
For consistency with the other similar summaries:
```suggestion
        /// Initializes a new instance of the <see cref="GZipStream"/> class by using the specified stream, compression options, and optionally leaves the stream open.
```